src/components/Tabs.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const selectedTab = tabs.find(tab => tab.id === tabId) || null;
+
+  return (
+    <>
+      {tabs.some(tab => tab.id === tabId) || tabId === undefined
```

Comment:
it's not necessary to use `some` method here because you simply can use your `selectedTab` variable.